.github/workflows/ci.yml

```
@@ -234,4 +236,85 @@ jobs:
+        name: coverage-report
+        path: coverage/
+
+    - name: Comment coverage report on PR
+      if: github.event_name == 'pull_request'
+      uses: actions/github-script@v7
+      continue-on-error: true
+      with:
+        script: |
+          const fs = require('fs');
+          const coveragePath = 'coverage/lcov-report/index.html';
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)